Node.js/GettingStarted/azure_batch_client.js

```diff
@@ -0,0 +1,196 @@
+// Initializing node.js modules
+
+var batch = require('azure-batch');
+var https = require('https');
+
+var fs = require('fs');
+var uuid = require('node-uuid');
+
+var accountName = '<azure-batch-account-name>';
+var accountKey = '<account-key-downloaded>';
+var accountUrl = '<account-url>'
+// Setting up Job configuration along with preparation task
```

Comment:
This is not really 'setting up the job configuration'  - just creating a named constant for the job ID.